### DISCUSSION TITLE
Automation for Creating Assets

### DISCUSSION THREAD

**Discussion by tim-hafner** [snipe/snipe-it#11569] [General]
Hey there,
we are missing the following feature:

Creating Assets, assigning cables (component) to the assets, automatically lower the quantity of the cables when asset is created and cables are assigned

In common, when we put a new System into a rack, we use always 2 network cables and 1 power cable.
It would be great if there is a feature which do this.
Does anybody know if there is such a function or added the function?

**Comment by Asyraf777** [comment]
Hi, if not mistaken in the latest version 6.0.8 already got the "predefined kit" feature. hope this will help a little

**Comment by tim-hafner** [comment]
Hi,
I saw the option of predefined kits, but just for users.
My request would be to use these kits for systems like Servers and their components.

**Reply by nicokempe** [reply]
You mean like with this plus button?
![image](https://user-images.githubusercontent.com/50241630/183635184-db04e08f-8182-4c57-bcef-8ddf95fc0f4f.png)

**Comment by tim-hafner** [comment]
Kind of. Just a feature, where we create a new asset and put a component like a SSD or some cables to the created server asset. The amount of components should be decreased when assigning to a server.
